standardguess

```diff
@@ -0,0 +1,382 @@
+#!/bin/bash
+#checks to see if a video's properties are consistent with a set of standards and if so it assigns expected values to other properties if they are undefined or differ from expectations 
+
+SCRIPTDIR=$(dirname $(which "${0}"))
```

Comment:
worth quoting the $() layer too, such as `SCRIPTDIR="$(dirname $(which "${0}")")`